app/routes/settings/email-preferences.js

```diff
@@ -7,19 +7,6 @@ export default Route.extend({
     return this.l10n.t('Email Preferences');
   },
   model() {
-    return [{
-      name             : 'Techtoma',
-      role             : 'Organiser',
-      isNewPaper       : true,
-      isChangeSchedule : true,
-      isNewEvent       : true
-    },
-    {
-      name             : 'Open Summit 2017',
-      role             : 'Organiser',
-      isNewPaper       : false,
-      isChangeSchedule : false,
-      isNewEvent       : false
-    }];
+    console.log(this.get('store').query('email-notification', {}));
```

Comment:
The relationship has to be queried via the user model. Look at this logically, if you just query `email-notification` from where will ember get the correct user ID ? It cannot. 

@sumedh123 please read https://guides.emberjs.com/v2.14.0/models/relationships more clearly.